pkg/minikube/bootstrapper/fake_runner.go

```diff
@@ -103,3 +103,11 @@ func (f *FakeCommandRunner) GetFileToContents(fpath string) (string, error) {
 func (f *FakeCommandRunner) SetFileToContents(fileToContents map[string]string) {
 	f.fileToContents.Store(fileToContents)
 }
+
+func (f *FakeCommandRunner) DumpMaps(w io.Writer) {
```

Comment:
I just added this as a nice debugging function for the fake runner, since I didn't want to actually make cmdMap and fileMap public, but it might be helpful to dump their contents on test failures.